DylibBinder/DBFunc.cs

```diff
@@ -32,7 +32,7 @@ internal class DBFunc : IAssociatedTypes {
 		public DBGenericParameters GenericParameters { get; }
 		public DBAssociatedTypes AssociatedTypes { get; } = new DBAssociatedTypes ();
 
-		public DBFunc (TLFunction tlf, bool isMutating = false, bool isProtocol = false)
+		public DBFunc (TLFunction tlf, bool isMutating = false, bool isProtocol = false, bool isGlobal = false)
```

Comment:
This function shouldn't need an `isGlobal` flag. You should get the same result from doing `tlf.IsTopLevelFunction`.
The `TLFunction` type is a holder for functions defined in a dylib. Basically, the `TL` stands for top level, but what happens when you get a member function? Then the member is defined on a class and the `TLFunction` will have a class associated with it and with a nesting name length > 0. This predicate checks that for you.